iq_events.info.yml

```diff
@@ -2,7 +2,7 @@ name: IQ Events module
 description: Provides functionality for creating and managing events.
 package: Custom
 type: module
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: 8.8 - 9.4
```

Comment:
Why do we have to change it at all? I'm confused..

Reply:
This should read as "we checked deprecations and warnings up to version 9.4"
^9 promises more than that. This is a non-critical suggestion; deprecations will only become critical when moving from 8 to 9 or 10

And this PR effectively did get rid of deprecation warnings making it more future-compatible in the iq_group dependency (see https://github.com/iqual-ch/iq_group/pull/14)

Do we need to change this and block ourselves from upgrading to e.g. 9.5? Yes, if we want to force smaller incremental changesets rather than bigger ones.

Reply:
And most importantly :-)
This came in via https://support-iqual.atlassian.net/browse/US-869

Reply:
Okay, didn't know about this effect. So, good to go for me :)